apps/designer/components/Buttons/HandleToggle.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const HandleToggle = ({
+  toggleButton,
+  myToggleButton,
+}: {
+  toggleButton: boolean;
```

Comment:
Use better names. For boolean state variables use names like `isToggled`. For handlers, use names like `handleClick` for methods. This is the conventional way to name such variables